### PR TITLE
gui(sell): creation of sell page, change in eslint

### PR DESCRIPTION
**Pull request by AtoLrn**
# Description

URL : http://localhost:3000/seller/sell
Création de la page de vente des objets, premier jets des models et modifications dans l'eslint pour retirer certaines erreurs

Fixes # (issue)

## Type of change

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] It runs and the features works

image

### CODE REVIEW

**Review by lealaroze — APPROVED** [review]
👍😍

File: src/pages/seller/sell.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+	palette: {
+		primary:{
+			main: '#E6BC17'
+		},
+		secondary:{
+			main: '#E6BC17'
+		},
+		type: 'dark'
+	}
+})
```
Comment (gelven4sec) [inline src/pages/seller/sell.js:18]: T'avais pas créé un theme que tu pouvais importer ?

Comment (AtoLrn) [reply src/pages/seller/sell.js:18]: Fais dans une autre branche :0

**Review by gelven4sec — APPROVED** [review]
(☞ﾟヮﾟ)☞

**Pull request merged**